src/cart/crt_swim.c

```diff
@@ -1069,7 +1075,7 @@ void crt_swim_accommodate(void)
 		else if (average > max_timeout)
 			average = max_timeout;
 
-		if (average != ping_timeout) {
+		if ((average >> 5) != (ping_timeout >> 5)) {
```

Comment:
what is this magic 5 shift?

Reply:
Just divide by 32 to skip small changes. Shift is much faster rather divide.

Reply:
Can you add a comment above it to explain this?

Reply:
sure,